xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/objects/classes/ComputedFieldClass.java

```diff
@@ -90,6 +90,47 @@ public String getScript()
         return sValue;
     }
 
+    /**
+     * Computes and returns the raw value of this property for a given object.
+     * @param name property name
+     * @param prefix prefix to be added
+     * @param object object for which the property value has to get computed
+     * @param context current context
+     * @return the computed property value
+     */
+    public String getComputedValue(String name, String prefix, BaseCollection object, XWikiContext context) {
+        String script = getScript();
+
+        try {
+            ScriptContext scontext = Utils.getComponent(ScriptContextManager.class).getCurrentScriptContext();
+            scontext.setAttribute("name", name, ScriptContext.ENGINE_SCOPE);
+            scontext.setAttribute("prefix", prefix, ScriptContext.ENGINE_SCOPE);
+            scontext.setAttribute("object", new com.xpn.xwiki.api.Object((BaseObject) object, context),
+                    ScriptContext.ENGINE_SCOPE);
+
+            XWikiDocument classDocument = object.getXClass(context).getOwnerDocument();
+
+            return renderContentInContext(script, classDocument.getSyntax().toIdString(),
+                    classDocument.getAuthorReference(), classDocument.getDocumentReference(), context);
+
+        } catch (Exception e) {
+            return e.getMessage();
+        }
+    }
+
+    /**
+     * Appends the raw computed property value of this object to the passed buffer.
+     * @param buffer buffer to which the property value has to get appended
+     * @param name property name
+     * @param prefix prefix to be added
+     * @param object object for which the property value has to get computed
+     * @param context current context
+     */
+    public void getComputedValue(StringBuffer buffer, String name, String prefix, BaseCollection object,
+            XWikiContext context) {
```

Comment:
Codestyle.

Reply:
This method was removed, taking into account Marius' suggestion indeed.